### PR TITLE
Docs: Add disconnect value feature documentation

### LINKED ISSUE #70282: docs: Add documentation for disconnect feature
Add documentation for new disconnect feature

This feature is available in three panels:
 
[Time series](https://github.com/grafana/grafana/pull/70097)
[State timeline](https://github.com/grafana/grafana/pull/70610)
[Trend](https://github.com/grafana/grafana/pull/70616)

### PR DESCRIPTION
**Pull request by nmarrs**
Add disconnect value feature documentation to [time series docs](https://grafana.com/docs/grafana/latest/panels-visualizations/visualizations/time-series/) and [state timeline docs](https://grafana.com/docs/grafana/latest/panels-visualizations/visualizations/state-timeline/). [Trend panel docs](https://grafana.com/docs/grafana/latest/panels-visualizations/visualizations/trend/) link directly to time series.

Fixes https://github.com/grafana/grafana/issues/70282

### CODE REVIEW

**Review by imatwawana — COMMENTED** [review]
I've left a suggestion for rewording the description of the setting. Let me know what you think. Otherwise it looks good!

For the State timeline panel, this setting is under the **State timeline** panel settings, so once we have this content nailed down, you can just add it in that corresponding section in the docs.

File: docs/sources/shared/visualizations/disconnect-values.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+title: Disconnect values
+---
+
+### Disconnect values
+
+Choose whether to set a threshold above which values in the data should be disconnected.
+
+{{< figure src="/media/docs/grafana/screenshot-grafana-10-1-disconnect-values.png" max-width="750px" caption="Disconnect values toggle" >}}
```
Comment (imatwawana) [inline docs/sources/shared/visualizations/disconnect-values.md:9]: Tiny nit, i don't think we need the caption here. Seems like overkill. 

image

```suggestion
{{< figure src="/media/docs/grafana/screenshot-grafana-10-1-disconnect-values.png" max-width="750px" >}}
```

**Review by imatwawana — APPROVED** [review]
Looks great! I LOVE a shared file and want to implement these a lot more across visualizations, so well played. One tiny nit, but otherwise LGTM!

File: docs/sources/shared/visualizations/disconnect-values.md (hunk 1 above)
Comment (nmarrs) [reply docs/sources/shared/visualizations/disconnect-values.md:9]: Thanks for pointing this out :) I was under impression that we should always include captions for improved a11y 🤷‍♂️ (screen readers etc.) - I agree though especially for this image it is pretty redundant so I'm ok with taking it out 👍

**Pull request merged**